tools/setup.sh

```diff
@@ -4,7 +4,7 @@
 set -euxo pipefail
 
 # Install Node.js dependencies
-npm install -g aws-cdk@2.176.0
+npm install -g aws-cdk@2.1007.0
```

Comment:
these library updates are not related to AG-1754.  they should be done in a separate PR.

Reply:
I'll update the description as I could not run with the existing versions.